include/ssd-internal.h

```diff
@@ -109,9 +119,6 @@ struct ssd_part {
 	/* This part represented in scene graph */
 	struct wlr_scene_node *node;
 
-	/* Targeted geometry. May be NULL */
-	struct wlr_box *geometry;
-
```

Comment:
I removed `ssd_part.geometry` since now we calculate all the x/y/width/height of extent parts in `ssd_extents_update()`.